### ISSUE TITLE
Attach media and extract content from file

### ISSUE THREAD

**Issue by DominiqueMakowski** [Eomm/why-don-t-you-tweet#10]
Hi @Eomm and thanks, the action works like a charm ☺️ 

I had a few questions: 

- Is it possible to add a feature of attaching a media (image) to the tweet?
- I would like this action to tweet every time a new file of a certain name pattern has been pushed, and to pull some info from that file for the tweet's content (say I push a file named `new_post3.txt`, I would like to tweet with something like `new post! the title is {extract title}`. Would you know how to achieve something like that? Any leads are welcome :)

Thanks again

**Comment by Eomm** [comment]
Yeah sure, adding media is possible and we need to update this action to support it.
Would you like to send a PR?

>  Would you know how to achieve something like that?

I think you need to run some bash scripting to archive this.
I would search for inspiration like this action: https://github.com/tj-actions/changed-files

**Comment by DominiqueMakowski** [comment]
> Would you like to send a PR?

Unfortunately I have no idea whatsoever of how that would look like :)

> I think you need to run some bash scripting to archive this.

Thanks for the hint!

**Comment by DominiqueMakowski** [comment]
Regarding the content of the tweet, thanks to your help I managed to query and store the tweet content in a variable in the `env`, that I managed to print [here](https://github.com/RealityBending/RealityBending.github.io/blob/96c2f25ec9be2b42b8b2ae2c16e3f5614940d6f0/.github/workflows/tweet.yml#L69). However, When I try to populate the [tweet-message](rhttps://github.com/RealityBending/RealityBending.github.io/blob/96c2f25ec9be2b42b8b2ae2c16e3f5614940d6f0/.github/workflows/tweet.yml#L81-L82) field, the action tweets `$TWEET_MESSAGE` instead of the content of the variable. How can I specify to take the variable from the environment instead of taking it as a string? Thanks!

**Comment by Eomm** [comment]
I think you need to define that variable into the tweet run's env, as the twitter api key. 

For security reasons you need to do it, so the run can not access all your envs without your explicitly declaration
